src/spring-cloud/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+3.1.4
+---
+* The argument "--build-env" accepts key[=value] instead of json.
```

Comment:
May I ask will this lead to breaking change?

Reply:
I don't think it's a breaking change. There are two reasons: 1. The enterprise tier is in public preview, and there are few users. 2. This command is a new command, we even don't have a doc for this command.